Patches/Colony Leadership/WarriorLeadership.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Colony Leadership</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/HediffDef[defName="leader2"]/stages/li[1]/statOffsets</xpath>
+					<value>
+          						<statOffsets>
+         							<TradePriceImprovement>+0.05</TradePriceImprovement>
+       							<NegotiationAbility>+0.05</NegotiationAbility>
+       							<MoveSpeed>+0.10</MoveSpeed>
+       							<MeleeHitChance>+0.05</MeleeHitChance>
+             							<MedicalTendQualityOffset>+0.05</MedicalTendQualityOffset>
+							<Suppressability>-0.12</Suppressability>
+          						</statOffsets>
+					</value>
+				</li>
+
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/HediffDef[defName="leader2"]/stages/li[1]/statOffsets</xpath>
+					<value>
+          						<statOffsets>
+       						<TradePriceImprovement>+0.10</TradePriceImprovement>

```

Comment:
I wouldn't include the TradePrice or Negotiation buffs, as the base mod doesn't give the Warrior Aura buffs of this nature.

Reply:
This is not part of the aura, this bonus is only for the leader himself, all leaders have trade-related bonuses.